RepoMan/RepoMan/Analysis/Scoring/ApprovalScorer.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Linq;
+using RepoMan.Analysis.ApprovalAnalyzers;
+using RepoMan.Repository;
+
+namespace RepoMan.Analysis.Scoring
+{
+    class ApprovalScorer :
+        PullRequestScorer
+    {
+        public const string Label = "ApprovalCount";
+        public override string Attribute => Label;
```

Comment:
The public constant is so I can refer to specific entries in a `Dictionary` later when building up a `Metrics` data file. The `Attribute` is to ensure that the shape is consistent. It seemed the least bad way to do it, but it's hardly elegant.